src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
```

Comment:
It's kinda unit tested by definition (a unit test would test that std:erf(z) - x ~ 0 for various values, which is exactly what the core of the function is doing), but fair enough.